package.json

```diff
@@ -156,6 +156,27 @@
     "test": "react-scripts test --env=./custom-test-env.cjs",
     "test:e2e": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run --record'"
   },
+  "jest": {
+    "collectCoverageFrom": 
+    [
+      "src/components/**/*.ts*", 
+      "src/hooks/**/*.ts*", 
+      "src/lib/hooks/**/*.ts*", 
+      "src/lib/state/**/*.ts*", 
+      "src/lib/utils/**/*.ts*", 
+      "src/pages/**/*.ts*",
+      "src/state/**/*.ts*",
+      "src/utils/**/*.ts*"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 70,
```

Comment:
These may be too high, considering where we are currently at.

Reply:
True, changed to 25

Reply:
I lowered them to just under where we are so our unit test check in the git automated checks don't block submissions.